deployments/mainnets/zora-mainnet.md

```diff
@@ -0,0 +1,29 @@
+# Zora Mainnet
+
+Chain Id: 7777777
+
+## LightAccountFactory
+
+| Version | Address                                      | Explorer                                                                                    | Salt                                                                 | Run                                                                             |
+| ------- | -------------------------------------------- | ------------------------------------------------------------------------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| v2.0.0  | `0x0000000000400CdFef5E2714E63d8040b700BC24` | [explorer](https://explorer.zora.energy/address/0x0000000000400CdFef5E2714E63d8040b700BC24) | `0x00000000000000000000000000000000000000005f1ffd9d31306e056bcc959b` | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1714101518.json) |
+| v1.1.0  | 0x00004EC70002a32400f8ae005A26081065620D20   | [explorer](https://explorer.zora.energy/address/0x00004EC70002a32400f8ae005A26081065620D20) | `0x4e59b44847b379578588920ca78fbf26c0b4956c3406f3bdc271500000c2f72f` | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1710446999.json) |
```

Comment:
Super small nit: the address column here is missing the code style via "`" chars.

(Side note: this is something an automated deployment generating script would fix).

Reply:
good eye!